infrastructure/helm/README.md

```diff
@@ -35,6 +35,82 @@ image:
 This image can be built with `docker build -t bailo .` in the root directory. This guide assumes the overrides file is
 called `local.yaml` in the `helm/bailo` folder.
 
+### Generate certs
+
+Basic certs can be in `backend/certs`
+
+1. `openssl genrsa -out key.pem 2048 && openssl req -new -x509 -key key.pem -out cert.pem -config san.cnf -extensions 'v3_req' -days 360`
+
+### minimal .local.yaml for OpenShift
+
+```yaml
+image:
+  frontendRepository: "image-registry-openshift-imagestreams"
+  frontendTag: tag
+  backendRepository: "image-registry-openshift-imagestreams"
+  backendTag: tag
+
+route:
+  enabled: true
+  appPublicRoute: openshift-route-url
+
+mongodb:
+  auth:
+    passwords:
+      - mongodb-password
+    usernames:
+      - mongodb-user
+
+openshift:
+  namespace: project-name
+```
+
+### minimal .local.yaml for AWS
+
+```yaml
+image:
+  frontendRepository: "aws-elastic-container-registry"
+  frontendTag: tag
+  backendRepository: "aws-elastic-container-registry"
+  backendTag: tag
+
+ingress:
+  enabled: true
+  name: "bailo-ingress"
+  annotations:
+    kubernetes.io/ingress.class: alb
+    alb.ingress.kubernetes.io/scheme: internet-facing
+    alb.ingress.kubernetes.io/target-type: instance
+  fqdn: "*.amazonaws.com"
+
+aws:
+  enabled: true
+
+mongodb:
+  persistence:
+    enabled: false
```

Comment:
With persistence false are we not keeping this data around?

Reply:
it's to do with the aws storage works. it would not kill the storage however will not try to create one either.

Reply:
Could we add a comment about this on the helm chart, as it seems not as one would expect.

Reply:
added to value.yaml